src/customFunctions.gs

```diff
@@ -170,7 +176,22 @@ function JST_search(JQL, Fields, Limit, StartAt) {
  * @customfunction
  */
 function JST_formatDuration(Seconds) {
+  customFunctionAllowed();
+
   Seconds = parseInt(Seconds) || 0;
 
   return formatTimeDiff(Seconds);
 }
+
+/**
+ * @desc Check if custom function is enabled by user otherwise throws new Error.
+ * @throws Error
+ * @return Void
+ */
+function customFunctionAllowed() {
```

Comment:
private function should be called customFunctionAllowed_ 
Not a show stopper as it doesnt introduce any security issues.

Reply:
Yes your right. I was thinking about a more clear definition for such. As not everything reallz must be private in terms of accessibility to the user environment. I think it would be enough to declare methods as private (Google private with tailing "_") only when the method has critical function or should never be executed without context.
What do you think.

The above method as example, for my understanding, would be fine to be executed by different users scripts (public) as it simply tells if a feature is enabled or not.

Reply:
I guess its just a matter of consistency - I know there are still alot of functions that should be private which don't have the underscore.
Agree this doesnt matter if its used - its not a security risk.
My opinion (non blocking) is that if the function is in the global scope (i.e. its just defined as a function in a source code and is not wrapped in an object) - we should add the underscore.
One day someone may add something to the function (or another one) which means it shouldnt be called. By setting the consistent approach I think it will benefit the code base and reduce the possibility of issues creeping in.